lib/compact_index/versions_file.rb

```diff
@@ -15,80 +15,34 @@ def contents(gems = nil, args = {})
 
       raise ArgumentError, "Unknown options: #{args.keys.join(", ")}" unless args.empty?
 
-      out = File.read(@path)
-      out << parse_gems(gems) if gems
-      out
+      File.read(@path).tap do |out|
+        out << gem_lines(gems) if gems
+      end
     end
 
     def updated_at
       created_at_header(@path) || Time.at(0).to_datetime
     end
 
-    def update_with(gems)
```

Comment:
Computers will always update the file. It will only be created one when decided by humans. This decision is done based on web stats. I think this rake task should actually be update only, since we already have one versions.list under git. Was done this way because we didn't had versions.list under git in the past.